spectrum.go

```diff
@@ -48,3 +52,5 @@ func TX_BOLD(textobjc string) string { return "\u001b[1m" + textobjc + "\u001b[0
 func TX_UNDERLINE(textobjc string) string { return "\u001b[4m" + textobjc + "\u001b[0m" }
 
 func TX_INVERTED(textobjc string) string { return "\u001b[7m" + textobjc + "\u001b[0m" }
+
+func HD_SUCCESS(textobjc string) string { return FR_GREEN(TX_BOLD("[SUCCESS] " + strings.ToUpper(textobjc))) }
```

Comment:
We do not need the entire success message to be in caps. Only the `SUCCESS` part will do.

Reply:
I fixed it !